### PR TITLE
Add BXN chains (testnet and mainnet)

### PR DESCRIPTION
**Pull request by BlackFortGroup**

### CODE REVIEW

File: _data/chains/eip155-4777.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "nativeCurrency": {
+    "name": "BlackFort Testnet Token",
+    "symbol": "TBXN",
+    "decimals": 18
+  },
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://blackfort.exchange",
+  "shortName": "TBXN",
+  "chainId": 4777,
+  "networkId": 477,
```
Comment (ligi) [inline _data/chains/eip155-4777.json:17]: typo?

File: _data/chains/eip155-4777.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "BlackFort Exchange Network Testnet",
+  "chain": "TBXN",
+  "rpc": [
+    "https://testnet.blackfort.network/rpc",
```
Comment (ligi) [inline _data/chains/eip155-4777.json:5]:
```suggestion
    "https://testnet.blackfort.network/rpc"
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-4777.json (hunk 1 above)
Comment (BlackFortGroup) [reply _data/chains/eip155-4777.json]: Yes, resolves in the last commit

**Review by Mey1212 — APPROVED** [review]

**Review by ligi — APPROVED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Review by 13364174888 — APPROVED** [review]

**Review by ligi — APPROVED** [review]

**Review by 13364174888 — COMMENTED** [review]
好

**Comment by Mey1212** [comment]
i dont understand chinese

**Pull request merged**